components/NavBar/index.js

```diff
@@ -0,0 +1,83 @@
+import Link from "next/link";
+import styled from "styled-components";
+import { AiFillHome } from "react-icons/ai";
+import { useState } from "react";
+import { AiOutlineHome } from "react-icons/ai";
+import { RiCalendarTodoFill } from "react-icons/ri";
+import { RiCalendarTodoLine } from "react-icons/ri";
+import { BsFilePersonFill } from "react-icons/bs";
+import { BsFilePerson } from "react-icons/bs";
```

Comment:
```suggestion
import { useState } from "react";
import { AiFillHome, AiOutlineHome } from "react-icons/ai";
import { RiCalendarTodoFill, RiCalendarTodoLine  } from "react-icons/ri";
import { BsFilePersonFill, BsFilePerson  } from "react-icons/bs";
```
You can import several things as the same time as long as they come from the same place =)